<?xml version="1.0" ?>
<launch>	
	<arg name="respawn" default="false" />	
	<param name="source_identifier" type="str" value="/stereo/right/image_raw" />
		<node name="right_blob_detector" pkg="color_blob_detector" type="hue_detector" respawn="$(arg respawn)" output="screen"/>
</launch>
